app/common/patient/models/patient.js

```diff
@@ -16,6 +16,15 @@ angular.module('common.patient')
                 this.birthdate = age.calculateBirthDate(this.age);
             };
 
+            var calculateBirthdateEstimated = function (value) {
+                if (value === 'age') {
```

Comment:
@ningosi the argument of the param `value` is hardcoded in the view `ng-blur="patient.calculateBirthdateEstimated('age')"`